### PR TITLE
Fix assert type

### PR DESCRIPTION
**Pull request by fiplox**
Here is a fix mentioned in #28 by @Marobax

### CODE REVIEW

**Comment by darnuria** [comment]
Look like it's ok! @fiplox did you mind adding @marobax as co-author and amend + push force? 

doc: https://docs.github.com/en/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors

**Comment by fiplox** [comment]
Ok, it did not do the trick. 
@Marobax, I'll need your email address (public one from github will be ok) to add you as co-author :)

**Comment by Marobax** [comment]
@fiplox I just made my email address public. Here on my [profile](https://github.com/Marobax) :)

**Comment by fiplox** [comment]
Done, @Marobax is in co-authors.
Now, if It is ok for @noalien it can be merged :)

**Comment by darnuria** [comment]
@fiplox would you authorize me to squash into one commit this fix? I am mostly against rebase/squash but 3 commit for one line fix is little off ahah.

If you prefer you can rebase it against upstream `master` and meld these two commit into one (with max as co-author).

**Comment by darnuria** [comment]
(did a save of your actual pr in case you're lost with git) ;)

**Comment by fiplox** [comment]
Yeah, was a little bit lost, but actually I had everything done but wasn't sure xD

File: lib_src/GL4D/gl4du.c [hunk 1]
```diff
@@ -1297,7 +1297,7 @@ void gl4duMultMatrixf(const GLfloat * matrix) {
 void gl4duMultMatrixd(const GLdouble * matrix) {
   GLdouble * mat, cpy[16];
   assert(_gl4dCurMatrix);
-  assert(_gl4dCurMatrix->type == GL_FLOAT);
+  assert(_gl4dCurMatrix->type == GL_DOUBLE);
```
Comment (darnuria) [inline lib_src/GL4D/gl4du.c:1300]: C'est bon pour moi! ^_^

**Review by darnuria — APPROVED** [review]

**Comment by darnuria** [comment]
Compile chez moi ok go.

**Pull request merged**